### ISSUE TITLE
CPython 3.13 support (tracker issue)

### ISSUE THREAD

**Issue by neutrinoceros** [yt-project/yt#4931]
This issue is to plan and keep track on the effort to add support for CPython 3.13 .
See [PEP 719](https://peps.python.org/pep-0719/) for the release schedule.
At the time of opening, version `3.13.0b2` is available, and NumPy is already publishing nightly wheels for it.

Here's the chronological task list to complete before we can declare support for it.
- [x] wait for our core dependencies to publish nightly builds, or be tested
	- [x] numpy
	- [x] matplotlib https://github.com/matplotlib/matplotlib/pull/28293
      - [x] Pillow
      - [x] contourpy
	- [x] unyt https://github.com/yt-project/unyt/pull/507 (not a real requirement: unyt 3.0.3 appear to be compatible already)
	- [x] cmyt https://github.com/yt-project/cmyt/pull/178
	- [x] ewah-bool-utils https://github.com/yt-project/ewah_bool_utils/pull/108
- [x] ~~https://github.com/yt-project/yt/pull/4958~~ replaced by #5021 
- [x] pass all tests on ubuntu
- [x] test `cp313` wheels builds for all platform in CI (but don't publish them yet)
	- [x] this is helped by #4949 
	- [x] ... and #4971
	- [x] #4973
- [x] enable publication to PyPI 
	- [x] requires ABI stability, i.e., CPython 3.13.0rc1
	- [x] #4973
- [ ] #5021

Supporting the experimental free-threading (a.k.a "nogil") build is a non-goal for this issue.
Relevant PRs are to be linked in this message.

**Comment by chrishavlin** [comment]
updated the checklist above to reflect 5021 replacing 4958.
